src/frontend/js/modules/Player.js

```diff
@@ -64,6 +64,10 @@ define(function(require) {
     globalShortcut.register('CmdOrCtrl+Down', () => {
       this.setVolume('down');
     });
+ 
+    globalShortcut.register('Escape', () => {
+      this._player.exitFullscreen();
+    });
```

Comment:
to be more specific, it should be something like this (Without testing, please make sure each API does work here)
```javascript
  globalShortcut.register(Escape, () => {
    this.ready().then(() => {
      this._player.exitFullscreen();
    });
  });
```